spring-cloud-pivotal-core/src/main/java/org/springframework/cloud/pivotal/service/common/GemfireServiceInfo.java

```diff
@@ -0,0 +1,65 @@
+package org.springframework.cloud.pivotal.service.common;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.cloud.service.BaseServiceInfo;
+import org.springframework.cloud.service.ServiceInfo.ServiceLabel;
+
+/**
+ * 
+ * Information to access Gemfire services
+ * 
+ * @author Vinicius Carvalho
+ *
+ */
+@ServiceLabel("gemfire")
+public class GemfireServiceInfo extends BaseServiceInfo {
+	final Pattern p = Pattern.compile("(.*)\\[(\\d*)\\]");
+	private URI[] locators;
+	private Map<String, Object> credentials;
+
+	public GemfireServiceInfo(String id, Map<String, Object> credentials) {
+		super(id);
+		this.credentials = credentials;
+		List<String> locatorList = (List<String>) credentials.get("locators");
+		this.locators = new URI[locatorList.size()];
+		for (int i = 0; i < locatorList.size(); i++) {
+			locators[i] = parseLocators(locatorList.get(i));
+		}
+	}
+
+	URI parseLocators(String locator) throws IllegalArgumentException {
```

Comment:
I assume `host[port]` is a GF syntax, and what the Pivotal broker provides in credentials. Would it make sense to support `locators: ["locator://host:port", "locator://host:port"]` also? Just wondering, since the URL syntax is what we are storing for later use anyway.